src/utils/mod.rs

```diff
@@ -9,6 +9,8 @@
 //! type name. Unions are converted to Rust `enum`s by dropping the `TPMU` qualifier and appending
 //! `Union`.
 pub mod algorithm_specifiers;
+pub mod tags;
+pub mod tickets;
```

Comment:
💯 nice, I think we'd do well to start splitting the rest of this file out, it's a mess of random types and utils for different things at the moment.